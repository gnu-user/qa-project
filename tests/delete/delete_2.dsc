TITLE
    Delete User Input
DESCRIPTION
    Run the login transaction using an admin account. Initiate the delete 
    transaction, and enter an existing, valid username when prompted. Verify 
    that the username entered is accepted.
DEPENDENCIES
    delete_1
