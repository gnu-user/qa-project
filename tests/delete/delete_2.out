Welcome.
> 
Enter username: 
Username accepted.
Available tickets file read successfully.
>
Enter the username to delete:
User deleted successfully.